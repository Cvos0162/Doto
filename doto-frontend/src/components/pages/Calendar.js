import React from "react";

const Calender = () => {
    return (
        <div className="App">
            <header className="App-header">
                <div>Calender</div>
            </header>
        </div>
    );
};

export default Calender;
