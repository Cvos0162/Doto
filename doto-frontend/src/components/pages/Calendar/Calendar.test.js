import React from "react";
import ReactDOM from "react-dom";
import { mount } from "enzyme";
import Calendar from "./Calendar";

describe("<Calendar /> component being rendered", () => {
    let subject;

    beforeEach(() => {
        subject = mount(<Calendar />);
    });

    afterEach(() => {
        subject.unmount();
    });

    it("Calendar component rendered without crashing", () => {
        const div = document.createElement("div");
        ReactDOM.render(<Calendar />, div);
    });
});
