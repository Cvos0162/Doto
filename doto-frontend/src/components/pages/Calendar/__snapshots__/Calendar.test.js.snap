// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Calendar /> component being rendered Make sure render matches snapshot 1`] = `ReactWrapper {}`;
